modules/components/src/Aggs/RangeAgg.js

```diff
@@ -170,6 +179,18 @@ class RangeAgg extends Component {
               />
               <RangeLabel isLeft>{this.formatRangeLabel(min)}</RangeLabel>
               <RangeLabel>{this.formatRangeLabel(max)}</RangeLabel>
+              <span
+                id={getLabelId(displayName)}
+                css={`
+                  position: absolute;
+                  height: 0;
+                  width: 0;
+                  top: -9999px;
+                  left: -9999px;
```

Comment:
@joneubank brought up a great point about potential weird scrolling behaviour. 

We don't really have any specific problems in mind, but before merging, can we get a quick check of this in Chrome, FF, Safari and Edge to make sure there's no strange side effect of this? :)

Just want to be extra careful in a shared library ;)

Reply:
Just tested in all of those browsers on the Mac, haven't seen any funkiness, looks just like normal. No unexpected scrolling behaviour. Don't have access to a Windows machine/VM on my OICR laptop so I haven't been able to test Windows browsers.

However, if you want we can eliminate the visually hidden element altogether and just point the aria-labelledby at an existing element. It won't be as descriptive but it'll satisfy the scanner

Reply:
Just completed re-testing using Windows via BrowserStack -- all good, no weird issues to report :)

Reply:
Awesome! thank you for the diligence man!